modules/ocf/manifests/configbuilder.pp

```diff
@@ -0,0 +1,22 @@
+define ocf::configbuilder(
+  $layout = {},
+  $owner = 'root',
+  $group = 'root',
+  $mode = '0400',
+  $show_diff = false,
+  $force = true,
```

Comment:
I think adding something here to validate passwords would be good, because we currently validate redis (and other) passwords in [a bunch](https://github.com/ocf/puppet/blob/785371b1461b7d69541fa95d92dc2e523502d0ad/modules/ocf_admin/manifests/create/redis.pp#L17) of [different](https://github.com/ocf/puppet/blob/785371b1461b7d69541fa95d92dc2e523502d0ad/modules/ocf_broker/manifests/init.pp#L18) [places](https://github.com/ocf/puppet/blob/785371b1461b7d69541fa95d92dc2e523502d0ad/modules/ocf_ocfweb/manifests/dev_config.pp#L7-L10). It would be better to have somewhere more unified to validate things coming from hiera if we have some restriction on what the value can actually be.